params/config.go

```diff
@@ -234,10 +236,13 @@ func (c *ChainConfig) IsSubnetEVM(blockTimestamp *big.Int) bool {
 
 // IsAllowList returns whether [blockTimestamp] is either equal to the AllowList fork block timestamp or greater.
 func (c *ChainConfig) IsAllowList(blockTimestamp *big.Int) bool {
+	if c.AllowListConfig == nil {
+		return false
+	}
 	return utils.IsForked(c.AllowListConfig.Timestamp(), blockTimestamp)
 }
 
-// IsIstanbul returns whether num is either equal to the Istanbul fork block or greater.
+// GetFeeConfig returns the *FeeConfig if it exists.
```

Comment:
Can we update the comment to reflect that it returns the default config otherwise?